presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPickTableLayout.java

```diff
@@ -96,6 +96,20 @@ public void doesNotFireIfTableScanNonDefaultConstraint()
                 .doesNotFire();
     }
 
+    @Test
+    public void doesNotFireIfRuleNotChangePlan()
+    {
+        tester().assertThat(pickTableLayout.pickTableLayoutForPredicate())
+                .on(p -> p.filter(expression("nationkey % 17 =  BIGINT '44'"),
```

Comment:
```
nationkey % 17 =  BIGINT '44' AND nationkey % 15 =  BIGINT '43'
```

Reply:
Thank you for investigation. Great find!